src/main/resources/assets/computercraft/lua/rom/programs/http/wget.lua

```diff
@@ -1,7 +1,7 @@
 
 local function printUsage()
     print( "Usage:" )
-    print( "wget [-r] <url> [filename]" )
+    print( "wget [run] <url> [filename]" )
```

Comment:
Usage here makes it look like I can do `wget run example.com test.lua` and `wget example.com`.

I know that the second will infer the file name but the first looks like it could be a bad input. The usage might need to examples: `wget run <url>` and `wget <URL> [filename]`